File: packtmasteringextjs/src/main/java/ch/rasc/packt/entity/Language.java
package ch.rasc.packt.entity;

import javax.persistence.Entity;

import ch.rasc.extclassgenerator.Model;

@Entity
@Model(value = "Packt.model.staticData.Language", disablePagingParameters = true,
		paging = true, createMethod = "languageService.create",
		readMethod = "languageService.read", updateMethod = "languageService.update",
		destroyMethod = "languageService.destroy")
public class Language extends SakilaBaseEntity {

	private String name;

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

}

File: packtmasteringextjs/src/main/webapp/app/model/staticData/Language.js
Ext.define("Packt.model.staticData.Language",
{
  extend : "Ext.data.Model",
  fields : [ {
    name : "name",
    type : "string"
  }, {
    name : "lastUpdate",
    type : "date",
    dateFormat : "c"
  }, {
    name : "id",
    type : "int",
    useNull : true,
    convert : null
  } ],
  proxy : {
    type : "direct",
    pageParam : undefined,
    startParam : undefined,
    limitParam : undefined,
    api : {
      read : "languageService.read",
      create : "languageService.create",
      update : "languageService.update",
      destroy : "languageService.destroy"
    },
    reader : {
      root : "records"
    }
  }
});